package.json

```diff
@@ -42,7 +42,9 @@
       "ecmaVersion": 2018,
       "sourceType": "script"
     },
-    "plugins": ["prettier"],
+    "plugins": [
+      "prettier"
+    ],
```

Comment:
This change introduces a nicer formatting but yet it is out of scope the project submitted.
Please restore the file its earlier state with `git checkout <commit-hash> -- ./path/to/file`.